src/components/body/Body.tsx

```diff
@@ -0,0 +1,42 @@
+//libs
+import { useCallback, useEffect, useState } from "react";
+
+//components
+import { SideBar } from "./components/sideBar/SideBar";
+import { ChatRoom } from "./components/chatRoom/ChatRoom";
+
+//hook
+import { useUserQuery } from "../../hooks/useUserQuery";
+import { useUserId } from "../../hooks/UserContext";
+
+//type
+import { User } from "../../types/types";
+
+//style
+import styles from "./Body.module.css";
+
+export const Body = () => {
+  const userId = useUserId();
+  const { userInfo } = useUserQuery(userId);
+
+  //activeTeamMate has the information about teamMate user currently chatting with
+  const [activeTeamMate, setActiveTeamMate] = useState<User>(userInfo);
```

Comment:
Fix this as discussed.

Shouldn't store selected user info in the state.
Instead, we should only store the ID of the selected user.